src/pages/Thanks/ThanksPage.vue

```diff
@@ -269,6 +271,15 @@ export default {
 		},
 	},
 	created() {
+		const filters = {
+			teamId: this.teamId,
+		};
+		const limit = 1;
+		fetchGoals(this.apollo, limit, filters)
```

Comment:
This will need to happen in the preFetch. Async functions in `created` won't complete on the server. The server immediately responds once `created` completes.

Reply:
Good call, a fundamental difference from CPS/nuxt.

Reply:
> add the teamsGoalsQuery within the Promise.all, then use readQuery on created to assign the goal

Yeah, I think that's the best way to do this in this component.